packages/react/src/index.ts

```diff
@@ -1,17 +1,10 @@
 export {
   createClient,
-  createClient as createWagmiClient,
-  Provider,
-  Provider as WagmiProvider,
+  ClientProvider,
+  ClientProvider as WagmiProvider,
```

Comment:
awesome! maybe we can deprecate usage of (and not document) `WagmiProvider` too?